source/dub/generators/sublimetext.d

```diff
@@ -86,7 +86,7 @@ private Json buildSystems(BuildPlatform buildPlatform, string workingDiretory =
 	string fileRegex;
 
 	if (buildPlatform.frontendVersion >= 2066 && buildPlatform.compiler == "dmd")
-		fileRegex = r"^(.+)\(([0-9]+)\,([0-9]+)\)\: (.*)$";
+		fileRegex = r"^(?:.*@)?(.+)\(([0-9]+)(?:\,([0-9]+))?\)\: (.*)$";
```

Comment:
I think it would be worth limiting the part before the `@` to identifier characters & dots only

Reply:
Ping @tofuninja , would you like to address the feedback ?